mysql-test/main/alter_table.test

```diff
@@ -1053,7 +1053,7 @@ ALTER TABLE t1 MODIFY COLUMN a SET('a1','a2','xx');
 ALTER TABLE t1 MODIFY COLUMN a SET('a1','a2','a0','xx');
 --echo # No copy: Add new to the end
 ALTER TABLE t1 MODIFY COLUMN a SET('a1','a2','a0','xx','a5','a6');
---echo # Copy: Numerical incrase (pack lenght)
+--echo # Copy: Numerical increase (pack length)
```

Comment:
reflected in alter_table.result, above which is good.

In mysql-test/suite/atomic/alter_table_{aria,rocksdb}.test a different alter_table.test file is imported.